app/views/practices/adoptions.html.erb

```diff
@@ -18,6 +19,9 @@
       <%= render partial: "shared/messages", locals: {small_text: false} %>
       <section class="usa-section padding-top-0 p0-bottom introduction">
         <h1 class="font-sans-2xl line-height-46px">Adoptions</h1>
+        <div class="margin-top-2 margin-bottom-2">
```

Comment:
We can remove this outer div because we added it to the `_practice_last_updated.html.erb` partial

Reply:
yep - got it.